[2021-03-09 18:33:53,369] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 18:33:55,602] [log.py:224] [log:log_response] [WARNING]- Not Found: /
[2021-03-09 18:33:55,603] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 179
[2021-03-09 18:33:55,725] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2021-03-09 18:34:00,121] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 302 0
[2021-03-09 18:34:00,151] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/login/?next=/admin/ HTTP/1.1" 200 3742
[2021-03-09 18:34:00,199] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/img/logo.png HTTP/1.1" 200 115997
[2021-03-09 18:34:00,241] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/img/bg.svg HTTP/1.1" 200 9547
[2021-03-09 18:34:01,655] [basehttp.py:157] [basehttp:log_message] [INFO]- "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2021-03-09 18:34:01,710] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 22265
[2021-03-09 18:34:01,762] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/css/index.css?_=3.2 HTTP/1.1" 200 7134
[2021-03-09 18:34:01,778] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/automatic/segment.js?_=2021.3 HTTP/1.1" 200 1186
[2021-03-09 18:34:01,785] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/js/cookie.js?_=2021.3 HTTP/1.1" 200 524
[2021-03-09 18:34:01,787] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/locale/en-us.js?_=2021.3 HTTP/1.1" 200 811
[2021-03-09 18:34:01,792] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/automatic/dicts.js?_=2021.3 HTTP/1.1" 200 6190
[2021-03-09 18:34:01,796] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/theme/theme.js?_=2021.3 HTTP/1.1" 200 4274
[2021-03-09 18:34:01,800] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/js/menu.js?_=2021.3 HTTP/1.1" 200 2089
[2021-03-09 18:34:01,801] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/locale/zh-hans.js?_=2021.3 HTTP/1.1" 200 1254
[2021-03-09 18:34:01,801] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/waves/waves.min.js?_=2021.3 HTTP/1.1" 200 6329
[2021-03-09 18:34:01,803] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/js/index.js?_=2021.3 HTTP/1.1" 200 20820
[2021-03-09 18:34:04,225] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/user/useranotherconfig/ HTTP/1.1" 200 23836
[2021-03-09 18:34:04,259] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/js/language.js?_=2.1.2 HTTP/1.1" 200 189
[2021-03-09 18:34:04,263] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/simpleui-x/locale/en-us.js?_=2.1.2 HTTP/1.1" 200 811
[2021-03-09 18:34:04,277] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7732
[2021-03-09 18:34:04,284] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/vendor/jquery/jquery.min.js HTTP/1.1" 200 89476
[2021-03-09 18:34:04,295] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
[2021-03-09 18:34:04,305] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/core.js HTTP/1.1" 200 5418
[2021-03-09 18:34:04,306] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/actions.min.js HTTP/1.1" 200 3237
[2021-03-09 18:34:04,308] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/prepopulate.min.js HTTP/1.1" 200 388
[2021-03-09 18:34:04,309] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/urlify.js HTTP/1.1" 200 8632
[2021-03-09 18:34:04,310] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1" 200 125266
[2021-03-09 18:34:04,310] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6078
[2021-03-09 18:34:04,455] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
[2021-03-09 18:34:04,994] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/user/usermodel/ HTTP/1.1" 200 23820
[2021-03-09 18:34:05,045] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7732
[2021-03-09 18:34:05,864] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/user/usermodel/add/ HTTP/1.1" 200 17925
[2021-03-09 18:34:05,889] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7732
[2021-03-09 18:34:05,905] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 492
[2021-03-09 18:34:05,908] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
[2021-03-09 18:34:05,908] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/admin/css/widgets.css HTTP/1.1" 200 10333
[2021-03-09 18:34:08,536] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/user/usermodel/ HTTP/1.1" 200 23820
[2021-03-09 18:34:08,583] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7732
[2021-03-09 18:35:19,699] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:27:11,637] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:27:17,043] [log.py:224] [log:log_response] [WARNING]- Not Found: /
[2021-03-09 20:27:17,044] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 179
[2021-03-09 20:27:17,166] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2021-03-09 20:27:41,925] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/ HTTP/1.1" 200 5998
[2021-03-09 20:27:41,997] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
[2021-03-09 20:27:42,007] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,008] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
[2021-03-09 20:27:42,012] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,013] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,016] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,017] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
[2021-03-09 20:27:42,019] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,020] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
[2021-03-09 20:27:42,022] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
[2021-03-09 20:27:42,036] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
[2021-03-09 20:27:43,910] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/ HTTP/1.1" 200 10943
[2021-03-09 20:27:43,954] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 18028
[2021-03-09 20:27:52,775] [basehttp.py:157] [basehttp:log_message] [INFO]- "POST /api/User/ HTTP/1.1" 201 9466
[2021-03-09 20:28:13,385] [log.py:224] [log:log_response] [WARNING]- Bad Request: /api/User/
[2021-03-09 20:28:13,386] [basehttp.py:157] [basehttp:log_message] [WARNING]- "POST /api/User/ HTTP/1.1" 400 9439
[2021-03-09 20:29:32,721] [autoreload.py:240] [autoreload:trigger_reload] [INFO]- /home/adminuser/PycharmProjects/NovelProject/apps/user/views.py changed, reloading.
[2021-03-09 20:29:33,180] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:32:40,900] [autoreload.py:240] [autoreload:trigger_reload] [INFO]- /home/adminuser/PycharmProjects/NovelProject/NovelProject/settings.py changed, reloading.
[2021-03-09 20:32:41,336] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:33:23,218] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:33:53,409] [log.py:224] [log:log_response] [WARNING]- Not Found: /api/user/1/PhoneCode/
[2021-03-09 20:33:53,409] [basehttp.py:157] [basehttp:log_message] [WARNING]- "POST /api/user/1/PhoneCode/ HTTP/1.1" 404 9604
[2021-03-09 20:34:03,035] [log.py:224] [log:log_response] [WARNING]- Method Not Allowed: /api/
[2021-03-09 20:34:03,036] [basehttp.py:157] [basehttp:log_message] [WARNING]- "POST /api/ HTTP/1.1" 405 46
[2021-03-09 20:34:07,263] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/ HTTP/1.1" 200 271
[2021-03-09 20:34:14,513] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/ HTTP/1.1" 200 11256
[2021-03-09 20:34:15,608] [log.py:224] [log:log_response] [WARNING]- Not Found: /favicon.ico
[2021-03-09 20:34:28,700] [log.py:224] [log:log_response] [WARNING]- Not Found: /api/user/1/
[2021-03-09 20:34:28,700] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET /api/user/1/ HTTP/1.1" 404 9573
[2021-03-09 20:34:39,417] [log.py:224] [log:log_response] [WARNING]- Not Found: /api/user/
[2021-03-09 20:34:39,418] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET /api/user/ HTTP/1.1" 404 9567
[2021-03-09 20:34:46,902] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:34:52,124] [log.py:224] [log:log_response] [WARNING]- Not Found: /
[2021-03-09 20:34:52,127] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET / HTTP/1.1" 404 2605
[2021-03-09 20:34:57,387] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/ HTTP/1.1" 200 5964
[2021-03-09 20:34:57,452] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
[2021-03-09 20:34:59,112] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/ HTTP/1.1" 200 11256
[2021-03-09 20:35:13,619] [basehttp.py:157] [basehttp:log_message] [INFO]- "POST /api/User/ HTTP/1.1" 200 9607
[2021-03-09 20:35:18,817] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/ HTTP/1.1" 200 11689
[2021-03-09 20:35:22,229] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/2/ HTTP/1.1" 200 11969
[2021-03-09 20:35:30,039] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/2/ HTTP/1.1" 200 220
[2021-03-09 20:35:33,426] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/1/ HTTP/1.1" 200 130
[2021-03-09 20:35:46,020] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /api/User/2/PhoneCode/
Traceback (most recent call last):
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/apps/user/views.py", line 100, in PhoneCode
    return Response(BaseResponse(Message=ResponseMessage.NullMessage.value).__str__(),
  File "/home/adminuser/PycharmProjects/NovelProject/BaseConfig/Response.py", line 14, in __str__
    if len(self.Phone) == 0:
TypeError: object of type 'NoneType' has no len()
[2021-03-09 20:35:46,021] [basehttp.py:157] [basehttp:log_message] [ERROR]- "GET /api/User/2/PhoneCode/ HTTP/1.1" 500 100943
[2021-03-09 20:36:46,901] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/1/ HTTP/1.1" 200 130
[2021-03-09 20:36:56,360] [log.py:224] [log:log_response] [WARNING]- Not Found: /api/User/1/set_phone_code/
[2021-03-09 20:36:56,360] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET /api/User/1/set_phone_code/ HTTP/1.1" 404 9618
[2021-03-09 20:37:11,657] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/2/PhoneCode/ HTTP/1.1" 200 48
[2021-03-09 20:37:22,627] [log.py:224] [log:log_response] [ERROR]- Internal Server Error: /api/User/2/PhoneCode/
Traceback (most recent call last):
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/adminuser/PycharmProjects/NovelProject/venv/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/adminuser/PycharmProjects/NovelProject/apps/user/views.py", line 100, in PhoneCode
    return Response(BaseResponse(Message=ResponseMessage.NullMessage.value).__str__(),
  File "/home/adminuser/PycharmProjects/NovelProject/BaseConfig/Response.py", line 14, in __str__
    if len(self.Phone) == 0:
TypeError: object of type 'NoneType' has no len()
[2021-03-09 20:37:22,628] [basehttp.py:157] [basehttp:log_message] [ERROR]- "GET /api/User/2/PhoneCode/ HTTP/1.1" 500 101080
[2021-03-09 20:37:45,653] [autoreload.py:240] [autoreload:trigger_reload] [INFO]- /home/adminuser/PycharmProjects/NovelProject/BaseConfig/Response.py changed, reloading.
[2021-03-09 20:37:46,104] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
[2021-03-09 20:37:46,453] [log.py:224] [log:log_response] [WARNING]- Unauthorized: /api/User/2/PhoneCode/
[2021-03-09 20:37:46,453] [basehttp.py:157] [basehttp:log_message] [WARNING]- "GET /api/User/2/PhoneCode/ HTTP/1.1" 401 9484
[2021-03-09 20:37:46,480] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 304 0
[2021-03-09 20:37:46,492] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
[2021-03-09 20:37:46,495] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
[2021-03-09 20:37:46,499] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
[2021-03-09 20:37:46,502] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
[2021-03-09 20:37:46,503] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
[2021-03-09 20:37:51,309] [basehttp.py:157] [basehttp:log_message] [INFO]- "GET /api/User/2/PhoneCode/ HTTP/1.1" 200 48
[2021-03-09 20:38:06,716] [log.py:224] [log:log_response] [WARNING]- Unauthorized: /api/User/2/PhoneCode/
[2021-03-09 20:38:06,717] [basehttp.py:157] [basehttp:log_message] [WARNING]- "POST /api/User/2/PhoneCode/ HTTP/1.1" 401 48
[2021-03-09 20:38:18,212] [log.py:224] [log:log_response] [WARNING]- Unauthorized: /api/User/2/PhoneCode/
[2021-03-09 20:38:18,213] [basehttp.py:157] [basehttp:log_message] [WARNING]- "POST /api/User/2/PhoneCode/ HTTP/1.1" 401 48
[2021-03-09 20:39:09,104] [autoreload.py:617] [autoreload:run_with_reloader] [INFO]- Watching for file changes with StatReloader
